languages/python/exercises/concept/comments/comments_test.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from commentpy import omit_String
```

Comment:
The name of the module is `comments` not `commentpy`; as per comment by @aimorris the **omit_String** name is not PEP-8 compliant.